tests/test_serializer.py

```diff
@@ -31,7 +32,8 @@ def test_invalid_serializer(self):
         serializer = self.Serializer(data={'char': 'abc'})
         assert not serializer.is_valid()
         assert serializer.validated_data == {}
-        assert serializer.errors == {'integer': ['This field is required.']}
+        assert serializer.errors['integer'].detail == ['This field is required.']
```

Comment:
Maybe I'm not understanding something here?
We simply can't change the return type of `errors` - it'll break stacks of existing projects.